astroquery/solarsystem/pds/core.py

```diff
@@ -0,0 +1,427 @@
+                ("fov_unit", planet.capitalize() + " radii"),
+                ("center", "body"),
+                ("center_body", planet.capitalize()),
+                ("center_ansa", planet_defaults[planet]["center_ansa"]),
+                ("center_ew", "east"),
+                ("center_ra", ""),
+                ("center_ra_type", "hours"),
+                ("center_dec", ""),
+                ("center_star", ""),
+                ("viewpoint", viewpoint),
+                ("observatory", observatory),
+                ("latitude", latitude),
+                ("longitude", longitude),
+                ("lon_dir", "east"),
+                ("altitude", altitude),
+                ("moons", planet_defaults[planet]["moons"]),
+                ("rings", planet_defaults[planet]["rings"]),
+                ("arcmodel", neptune_arcmodels[int(neptune_arcmodel)]),
+                ("extra_ra", ""),  # figure options below this line, can all be hardcoded and ignored
+                ("extra_ra_type", "hours"),
+                ("extra_dec", ""),
+                ("extra_name", ""),
+                ("title", ""),
+                ("labels", "Small (6 points)"),
+                ("moonpts", "0"),
+                ("blank", "No"),
+                ("opacity", "Transparent"),
+                ("peris", "None"),
+                ("peripts", "4"),
+                ("arcpts", "4"),
+                ("meridians", "Yes"),
+                ("output", "html"),
+            ]
+        )
+
+        # return request_payload if desired
+        if get_query_payload:
+            return request_payload
+
+        # query and parse
+        response = self._request(
+            "GET", self._url, params=request_payload, timeout=self.timeout, cache=cache
+        )
+
+        return response
+
+    def _parse_result(self, response, verbose=None):
+        """
+        Routine for parsing data from ring node
+
+        Parameters
+        ----------
+        self : RingNodeClass instance
+        response : list
+            raw response from server
+
+
+        Returns
+        -------
+        bodytable : `astropy.QTable`
+        ringtable : `astropy.QTable`
+        """
+        try:
+            self._last_query.remove_cache_file(self.cache_location)
+        except (FileNotFoundError, OSError):
+            # this is allowed: if `cache` was set to False, this
+            # won't be needed
+            pass
+
+        soup = BeautifulSoup(response.text, "html.parser")
+        text = soup.get_text()
+        textgroups = re.split(2*os.linesep+"|"+os.linesep+" "+os.linesep, text)  # need regex because some blank lines have spacebar and some do not
+        ringtable = None
+        for group in textgroups:
+            group = group.strip()
+
+            # input parameters. only thing needed is epoch
+            if group.startswith("Observation"):
+                epoch = group.splitlines()[0].split("e: ")[-1].strip()
```

Comment:
There is a method for splitting from the right:
```suggestion
                epoch = group.splitlines()[0].rsplit("e: ", 1)[1].strip()
```